packages/web/src/pages/earn/pool/[pool-path]/remove.tsx

```diff
@@ -24,15 +23,15 @@ export default function Earn() {
       { title: "Earn", path: "/earn" },
       {
         title:
-          breakpoint === DEVICE_TYPE.WEB
+        width >= 1150
```

Comment:
Declare a variable corresponding to 1150.

Reply:
done